float64 confidence
int64 topleft_x
int64 topleft_y
int64 bottomright_x
int64 bottomright_y
int16 id
string Class
